code_snippets/results/result.api-billing-usage-hosts.sh

```diff
@@ -0,0 +1,31 @@
+{
+  usage: [
+  {
+     "host_count": 7,
+     "container_count": 4,
+     "hour": "2017-07-24T00",
+     "apm_host_count": 8,
+     "agent_host_count": 9,
+     "apm_trace_count": 6,
```

Comment:
sorry for yet another change, but since we aren't billing for traces, we shouldn't include trace count either.